src/azurelinux/3.0/net8.0/webassembly/amd64/Dockerfile

```diff
@@ -18,15 +20,30 @@ RUN npm i -g typescript
 ENV EMSCRIPTEN_VERSION=3.1.34
 ENV EMSCRIPTEN_PATH=/usr/local/emscripten
 ENV EMSDK_PATH=/usr/local/emscripten/emsdk
+ENV NODE_VERSION_IN_EMSDK=15.14.0_64bit
 
 RUN mkdir ${EMSCRIPTEN_PATH} \
     && cd ${EMSCRIPTEN_PATH} \
     && git clone https://github.com/emscripten-core/emsdk.git ${EMSDK_PATH} \
     && cd ${EMSDK_PATH} \
     && git checkout ${EMSCRIPTEN_VERSION} \
+    # patch node version in emsdk_manifest.json
+    && sed -i 's/14\.18\.2/15\.14\.0/g' emsdk_manifest.json \
```

Comment:
you mean in a variable? we can't just pick any node version since it needs to be one that emsdk shipped.

Reply:
I mean use a regex to match on the existing version (not a hardcoded version but one that has a simple version pattern) defined in the JSON file and replace it with the new value, referencing an `ENV` value that could be defined.

Reply:
I'm not sure why that'd be necessary given that emsdk 3.1.34 will always ship with node 14.18.2 and as mentioned in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/1281 we can't upgrade the emsdk version.

Reply:
or do you mean the replacement version?

Reply:
I'm referring to both versions so that neither have to be hardcoded on this line.

> emsdk 3.1.34 will always ship with node 14.18.2

So when the emsdk version is updated, that requires knowing what the node version is and having to update this line with that value. My aim here is to reduce the amount of work needed to rev the versions.

Reply:
> So when the emsdk version is updated, that requires knowing what the node version is and having to update this line with that value. 

No, what I'm saying is the emsdk version is frozen for net8, we can't update it. And this logic is gone in net9+

Reply:
Ok, got it. Thanks